src/_test_/commentCounter.test.js

```diff
@@ -0,0 +1,32 @@
+/** @jest-environment jsdom */
```

Comment:
- [ ] Please do the same type of tests suggested for the movie counter here as mentioned by the previous reviewer. 🥸️